README.md

```diff
@@ -636,39 +664,43 @@ Note: Flags that use pid number as input should be used at last, if you are not
 
 #### Background Sync Jobs
 
-There are basically two ways to start a background, first one we already covered in the above section.
+There are basically two ways to start a background job, first one we already covered in the above section.
 
 It will indefinitely run until and unless the host machine is rebooted.
 
 Now, a systemd service service can also be created which will start sync job after boot.
 
-1.  Install the systemd unit file `sudo cp -v gsync.service /etc/systemd/system/`
+1.  To generate a systemd unit file, run the sync command with `--service service_name` at the end.
 
-1.  Open `/etc/systemd/system/gsync.service` with an editor of your choice and modify the following:
+    e.g: If `gsync foldername -d drive_folder --service myservice`
 
-    -   line 7, line 10 : User and ExecStart
+           where, myservice can be any name desired.
```

Comment:
Codacy found an issue: [[code-block-style] Code blocks should be fenced](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5767132)